draft-ietf-quic-manageability.md

```diff
@@ -584,19 +598,55 @@ Handshake RTT can be measured by adding the client-to-observer and
 observer-to-server RTT components together. This measurement necessarily
 includes any transport and application layer delay at both sides.
 
-The spin bit
-provides an additional method to measure intraflow per-flow RTT.  When a QUIC
-flow is sending at full rate (i.e., neither application nor flow control
-limited), the latency spin bit on short header packets changes value once
-per round-trip time (RTT). An on-path observer can observe the time difference
-between edges in the spin bit signal in a single direction to measure one
-sample of end-to-end RTT. Note that this measurement, as with passive RTT
-measurement for TCP, includes any transport protocol delay (e.g., delayed
-sending of acknowledgements) and/or application layer delay (e.g., waiting for
-a request to complete). It therefore provides devices on path a good
-instantaneous estimate of the RTT as experienced by the application. A simple
-linear smoothing or moving minimum filter can be applied to the stream of RTT
-information to get a more stable estimate.
+# Using the Spin Bit for Passive RTT Measurement {#spin-usage}
+
+The spin bit provides an additional method to measure intraflow per-flow RTT
+from observation points on the network path throughout the duration of a
+connection. Support of the spin bit is an optional feature in QUIC. Usage of
```

Comment:
You may want to clarify what "support" means. How about:
"Supporting the spin bit is an optional feature in QUIC.  That is, while its location is fixed in this version of QUIC, an endpoint can unilaterally choose to not support "spinning" the bit.  Use of the spin bit for RTT measurement by devices on path is only possible when both endpoints enable it."

Reply:
maybe "Endpoint participation in spin bit signaling is optional in QUIC." as the first sentence, but yep...

Reply:
Done